### PR TITLE
Staff Booking List

### PR DESCRIPTION
**Pull request by LongNguyenVu181**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/39853?issue_count=23&issue_position=12&next_issue_id=39852&prev_issue_id=39854

## WHAT
- Booking List Page for Staff

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by LongNguyenVu181** [comment]
ready

File: hotel/views.py [hunk 1]
```diff
@@ -205,3 +205,12 @@ def list_users(request):
         else:
             context = {'users': users}
         return render(request,'user/list-users.html',context)
+ 
+@login_required
+@permission_required('hotel.staff_bookng_list', raise_exception=True)
```
Comment (huongnt-2545) [inline hotel/views.py:210]: `staff_bookng_list` có gì sai sai nè?
ko check lại hở em

File: hotel/views.py [hunk 2]
```diff
@@ -205,3 +205,12 @@ def list_users(request):
...
+def list_bookings_staff(request):
+    bookings = Booking.objects.filter(status = 'waiting').all()
+    context = {
+        "bookings": bookings
+    }
+    return render(request,'user/staff-list-bookings.html',context)
```
Comment (huongnt-2545) [inline hotel/views.py:216]:
```suggestion
    return render(request, 'user/staff-list-bookings.html', context)
```

File: hotel/views.py [hunk 3]
```diff
@@ -205,3 +205,12 @@ def list_users(request):
         else:
             context = {'users': users}
         return render(request,'user/list-users.html',context)
+ 
+@login_required
+@permission_required('hotel.staff_bookng_list', raise_exception=True)
+def list_bookings_staff(request):
+    bookings = Booking.objects.filter(status = 'waiting').all()
```
Comment (huongnt-2545) [inline hotel/views.py:212]: Khai báo constant giá trị status

```suggestion
    bookings = Booking.objects.filter(status = 'waiting')
```

File: hotel/views.py (hunk 1 above)
Comment (LongNguyenVu181) [reply hotel/views.py]: Sorry chị em sửa rồi ạ :'(

**Comment by LongNguyenVu181** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**